supplementary/devices/index.rst

```diff
@@ -26,8 +26,8 @@ All of these should only be document in the exact same way here and **only an ov
    T00F
    titan
    vegeta
+   wt88047
    yuga
-   taoshan
```

Comment:
Don't remove other devices please?

Reply:
i never removed it

Reply:
You removed it in this commit, if it was an accident or not: https://github.com/Halium/docs/pull/69/commits/44f740c6e5a86fc2917ce60a9d40ac106ee66518